CHANGELOG.md

```diff
@@ -41,3 +40,6 @@ a variable that you can 'execute' or 'update'.
 - Adds Kubernete authentication
 - Optimize audit database and responses, for a significant improvement of performance.
 - `start` no longer fails to show Help information.
+
+[Unreleased]: https://github.com/cyberark/conjur-api-java/compare/v2.0.0...HEAD
+[2.0.0]: https://github.com/cyberark/conjur-api-java/compare/v1.1.0...v2.0.0
```

Comment:
Can add an entry for initial release:
```
## [1.1.0] - 2014-06-10
### Added
- Initial release
```
and add a link for this at the end:
```
[1.1.0]: https://github.com/cyberark/conjur-api-java/releases/tag/v1.1
```

Reply:
This is tricky because there seems to be 2 competing version histories. I think they target different versions of Conjur

Reply:
Thanks, I was told to skip initial releases initially, because of a bug in the parser. So most of the other CHANGELOGs don't have an initial release link either.